src/main.cpp

```diff
@@ -199,6 +220,13 @@ void loop() {
         settingsAppSwitcher.paginationEnable();
         mainAppSwitcher.registerApp(&settingsAppSwitcher);
 
+        // new tools screen
+        ToolsAppSwitcher.registerApp(new OswAppToolsApp());
+        ToolsAppSwitcher.registerApp(new OswAppToolsPage2App());
+        ToolsAppSwitcher.paginationEnable();
+        mainAppSwitcher.registerApp(&ToolsAppSwitcher);
```

Comment:
I see you've used the concept of the tools screen here (utilizing the app switcher), but I don't really get why you introduced the new classes for the tool "pages". These are dummy classes and their page it intended to be overwritten - right? So why not adapting a similar implementation like the other switchers? Like creating a tools-switcher and then registering other apps into it (instead of into the main switcher). You could also think about adding the concept of "pages" into the app switchers, but this is only an idea...

So, what do you want to do with this? :confused: :thinking: